projects/questionset-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -153,6 +154,10 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
         this.getFrameworkDetails(this.primaryCategoryDef);
       }
     });
+    this.editorService.readComment(this.collectionId).subscribe((res) => {
+      this.draftComment = res.result.comments[0].comment;
```

Comment:
for new question sets this will be the response from API.
```
{
    "comments": [
        {
            "comment": ""
        }
    ]
}
```
so it will not fail.